README.rst

```diff
@@ -157,7 +157,12 @@ Contributors
 	* `Patch #2 <https://github.com/softvar/json2html/pull/2>`_
 	* Added support for clubbing Array of Objects with same keys, more readable format.
 	* Added support for adding custom `table_attributes`.
-	* Better error message on bad call of 'convert'
+	* Better error message on bad call of 'convert'.
+	* Better error message on misbehaving input.
```

Comment:
Please see the latest commit - there's no such thing anymore, json2html is now very promiscuous and accepts anything you throw at it... I challenge you ;).